lib/alephant/logger/cloudwatch.rb

```diff
@@ -3,8 +3,8 @@
 module Alephant
   module Logger
     class CloudWatch
-      def initialize(namespace)
-        @namespace = namespace
+      def initialize(opts)
+        @defaults   = opts
```

Comment:
Should have clarified: suggested this with the intention of maintaining the default values that were being optionally assigned originally.  Something like a series of:

```
@defaults.merge { opts.fetch(:unit) { "Count" } }
```